client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -140,10 +149,30 @@ export const SeoForm = React.createClass( {
 	},
 
 	componentWillReceiveProps( nextProps ) {
-		const { selectedSite: prevSite, isFetchingSite } = this.props;
+		const { selectedSite: prevSite, isFetchingSite, translate } = this.props;
 		const { selectedSite: nextSite } = nextProps;
 		const { dirtyFields } = this.state;
 
+		// save success
+		if ( this.state.isSubmittingForm && nextProps.isSaveSuccess ) {
+			this.props.markSaved();
+			this.props.requestSiteSettings( nextProps.siteId );
+			this.refreshCustomTitles();
+			this.setState( { isSubmittingForm: false } );
+		}
+
+		// save error
+		if ( this.state.isSubmittingForm && nextProps.saveError ) {
+			this.setState( { isSubmittingForm: false } );
+			switch ( nextProps.saveError.error ) {
+				case 'invalid_ip':
+					notices.error( translate( 'One of your IP Addresses was invalid. Please, try again.' ) );
```

Comment:
This can be removed, folks. It is probably a remnant of the old boilerplate and the old implementation of the Jetpack Protect card that has an IP field. Let's just keep a generic error message here.